cloudigrade/config/settings/base.py

```diff
@@ -488,6 +488,8 @@ def __print(*args, **kwargs):
 CLOUDIGRADE_REQUEST_HEADER = "X-CLOUDIGRADE-REQUEST-ID"
 INSIGHTS_REQUEST_ID_HEADER = "HTTP_X_RH_INSIGHTS_REQUEST_ID"
 INSIGHTS_IDENTITY_HEADER = "HTTP_X_RH_IDENTITY"
+INSIGHTS_IDENTITY_ACCOUNT_HEADER = "HTTP_X_RH_IDENTITY_ACCOUNT"
+INSIGHTS_IDENTITY_ORG_ADMIN_HEADER = "HTTP_X_RH_IDENTITY_ORG_ADMIN"
```

Comment:
I think these new names looks too close to the real identity header name. It should be abundantly clear that we're (sometimes) paying attention to extra headers that fake or override the primary identity header.

Here's one possible suggestion that collapses the two into one (as mentioned elsewhere in my comments):

```suggestion
INTERNAL_FAKE_INSIGHTS_IDENTITY_HEADER = "HTTP_X_RH_INTERNAL_FAKE_IDENTITY"
```